src/types/amplitude-js/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'amplitude-js'
```

Comment:
I think you can add a `@types/amplitude-js` dependency instead of declaring the types manually